src/main/scala/viper/gobra/util/Bounds.scala

```diff
@@ -0,0 +1,20 @@
+package viper.gobra.util
+
+case class Bounds(lower: BigInt, upper: BigInt)
+
+case object Bounds {
+  val Int: Bounds = Bounds(-2147483648, 2147483647) // TODO: allow for changes according to the int size of the machine arch (either 32 or 64 bit)
```

Comment:
write an issue for this, so that it is not forgotten.

Reply:
Issue #95